deduplicator/src/bin/main.rs

```diff
@@ -41,17 +41,13 @@ struct Params {
     #[structopt(long, default_value = "addresses.db")]
     output_db: PathBuf,
 
-    /// Keep construction tables in the output database
+    /// Keep construction tables and the output database
     #[structopt(short, long)]
     keep: bool,
 
-    /// Output database as an OpenAddress-like CSV file
-    #[structopt(long)]
-    output_csv: Option<PathBuf>,
-
     /// Output database as an OpenAddress-like gzip CSV file
-    #[structopt(short, long)]
-    output_compressed_csv: Option<PathBuf>,
+    #[structopt(short, long, default_value = "deduplicated.csv.gz")]
+    output_csv: PathBuf,
```

Comment:
This is a breaking change for docker_mimir, right ?  

Please keep `output_compressed_csv` as a deprecated option, or prepare another PR in docker_mimir in order to apply this change consistently on the two repositories.

Reply:
Nice catch ! I reversed to the previous name for compatibility sake :)